lib/LaTeXML/Common/Dimension.pm

```diff
@@ -14,56 +14,92 @@ use strict;
 use warnings;
 use LaTeXML::Global;
 use LaTeXML::Common::Object;
+use LaTeXML::Common::Number;
 use base qw(LaTeXML::Common::Number);
 use base qw(Exporter);
-our @EXPORT = (qw(&Dimension));
+our @EXPORT = (qw(&Dimension &fixedformat  &fixpoint $UNITY));
 
 #======================================================================
 # Exported constructor.
 
+# Create a Dimension given either a float with unit OR a number in scaled points
 sub Dimension {
-  my ($scaledpoints) = @_;
-  return LaTeXML::Common::Dimension->new($scaledpoints); }
+  my ($spec) = @_;
+  return LaTeXML::Common::Dimension->new($spec); }
 
 #======================================================================
+sub _unit { return 'pt'; }
```

Comment:
Mostly because it was supposed to be secret. But you saw it anyway. Was trying to find the cleanest way to polymorph dimensions and mu by having a magic "mu" flag appear at the right times.